macosx/Controller.m

```diff
@@ -4000,73 +4117,84 @@ - (NSArray *) toolbarDefaultItemIdentifiers: (NSToolbar *) toolbar
 - (BOOL) validateToolbarItem: (NSToolbarItem *) toolbarItem
 {
     NSString * ident = [toolbarItem itemIdentifier];
-
-    //enable remove item
+    BOOL enabled = NO;
```

Comment:
Instead of goring this method's implementation by rewriting this with a single return point, rename the original method to `isToolbarItemValid`, and replace `validateToolbarItem` with:
```objc
BOOL enabled = [self isToolbarItemValid: toolbarItem];
[self syncToTouchBarForItem: ident withValue: enabled];
return enabled;
```